### PR TITLE
feat: add neb result command

### PR DESCRIPTION
**Pull request by DevAndromeda**
usage: `/neb symbol dob`

image

### CODE REVIEW

**Review by thenishantsapkota — APPROVED** [review]

File: src/utils/neb.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+            name: data[1],
+            creditHours: data[2],
+            grade: data[3]
+        });
+    });
+
+    return resultData;
+}
+
+export interface NEBSubjects {
```
Comment (dibashthapa) [inline src/utils/neb.ts:91]: if possible, add these typings in another folder

File: src/utils/neb.ts [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        .then((res) => res.text())
+        .then(scrape)
+        .catch(() => null);
+
+    if (!res) return null;
+    await redis.set(`neb::${dob}::${symbolNo}`, JSON.stringify(res));
+    return res;
+}
+
+function scrape(html: string) {
```
Comment (dibashthapa) [inline src/utils/neb.ts:44]: add this function in utils.

**Review by dibashthapa — APPROVED** [review]

File: src/utils/neb.ts (hunk 2 above)
Comment (DevAndromeda) [reply src/utils/neb.ts:44]: this function can only scrape neb results page so I think it's best to be inside neb file

Comment (dibashthapa) [reply src/utils/neb.ts:44]: hmm makes sense. Alright

**Review by TheLearneer — APPROVED** [review]
LGTM

**Comment by thenishantsapkota** [comment]
LGTM!

**Pull request merged**